src/routes/profile/%40[user]/+layout.svelte

```diff
@@ -8,16 +8,16 @@
 	// https://github.com/sveltejs/kit/issues/269
 	$: segments = $page.url.pathname.split('/');
 	$: is_favorites = segments.length === 4 && segments[3] === 'favorites';
-	$: is_user = $session.user && profile.username === $session.user.username;
+	$: is_user = $session.user && data.profile.username === $session.user.username;
 
 	let current_token;
 	async function toggle_following() {
 		const token = (current_token = {});
 
-		const { following, username } = profile;
+		const { following, username } = data.profile;
 
 		// optimistic UI
-		profile.following = !profile.following;
+      data.profile.following = !data.profile.following;
```

Comment:
can you change the spaces back to tabs on this line and the one below?

Reply:
I hope it's fixed now :)